media-fonts/font-misc-meltho/font-misc-meltho-1.0.3-r2.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+FONT_DIR="OTF"
+
+inherit xorg-3
+
+DESCRIPTION="X.Org Syriac fonts"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+
```

Comment:
This package, and `media-fonts/font-xfree86-type1` are the only ones I saw in a quick glance that didn't have `BDEPEND="x11-apps/bdftopcf"`. The likelihood of anyone actually avoiding this dependency is vanishingly small (they'd have to install none of the fonts that require it), so I'd prefer that we simplify all the font ebuilds and instead just add this dependency to the eclass directly.